src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -48,6 +48,53 @@
     <Reference Include="FSharp.Compiler.CodeDom">
       <HintPath>..\..\packages\FSharp.Compiler.CodeDom.1.0.0.1\lib\net40\FSharp.Compiler.CodeDom.dll</HintPath>
     </Reference>
+    <Reference Include="Newtonsoft.Json">
+      <HintPath>..\..\packages\Newtonsoft.Json.11.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
```

Comment:
@jonpryor do we have to add all this junk to the Mac installer?

Reply:
@dellis1972: if we need it at "runtime" -- app build time, packaging time, etc. -- then *yes*, we need to add this to the macOS installer.

Reply:
ok, will need to remember that for when we bump monodroid.